cmake/onnxruntime_unittests.cmake

```diff
@@ -502,6 +502,15 @@ if (onnxruntime_BUILD_SHARED_LIB)
   endif()
 endif()
 
+if (onnxruntime_BUILD_HOSTING)
+  #add_executable(hosting_tests "${ONNXRUNTIME_ROOT}/test/hosting/http_routes_tests.cc")
```

Comment:
Remove it

Reply:
Woops. It was greyed out so didn't see it.